crossfit/op/base.py

```diff
@@ -26,7 +26,10 @@ def __init__(self, pre=None, cols=False, keep_cols=None):
         self.pre = pre
         self.cols = cols
         self.keep_cols = keep_cols or []
-        self.worker_name = getattr(self.get_worker(), "name", 0)
+
+    @property
+    def worker_name(self):
+        return getattr(self.get_worker(), "name", 0)
```

Comment:
And historically it was returning zero because... `self` was constructed on client or..?

Reply:
Yeah exactly that